src/XrdAccSciTokens.cc

```diff
@@ -666,10 +735,13 @@ class XrdAccSciTokens : public XrdAccAuthorize
     uint64_t m_next_clean{0};
     XrdSysError m_log;
     AuthzBehavior m_authz_behavior{AuthzBehavior::PASSTHROUGH};
+    std::string m_cfg_file;
 
     static constexpr uint64_t m_expiry_secs = 60;
 };
 
+std::string      cfgSciTokens;
+
 extern "C" {
 
 XrdAccAuthorize *XrdAccAuthorizeObject(XrdSysLogger *lp,
```

Comment:
OK, this is the unction called when you are at the base of the food chain (i.e. the first plugin that is loaded. Hence, there is no pointer passed as there is no other acc object at this point. The code that originally existed in creating a default authorize object should be removed. If someone wanted a default authorize object they would specify:
ofs.authlib ++ libscitokens.so
if they didn't want the default authorization (i.e. they *just* want scitokens auth) then they would specify:
ofs.authlib libscitokens.so

In the "++" the framework automatically loads and configures default authorization and then load te scitokens plugin handing it a pointer to the previous plugin. In he non "++" case, the scitokens plugin is loaded as the base authorization object, it is essentially he default one. So, you should not call XrdAccDefaultAuthorizeObject()  that's already done for you if need be.